tests/test_suspend_resume/src/main.xc

```diff
@@ -0,0 +1,68 @@
+// Copyright 2016-2021 XMOS LIMITED.
+// This Software is subject to the terms of the XMOS Public Licence: Version 1.
+#include <xs1.h>
+#include <print.h>
+#include <stdio.h>
+#include "xud.h"
+#include "platform.h"
+#include "shared.h"
+
+#define XUD_EP_COUNT_OUT   5
+#define XUD_EP_COUNT_IN    5
+
+#ifndef PKT_LENGTH_START
+#define PKT_LENGTH_START 10
+#endif
+
+#ifndef PKT_LENGTH_END
+#define PKT_LENGTH_END 11
+#endif
+
+#ifndef TEST_EP_NUM
+#error
+#endif
+
+/* Endpoint type tables */
+XUD_EpType epTypeTableOut[XUD_EP_COUNT_OUT] = {XUD_EPTYPE_CTL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL};
+XUD_EpType epTypeTableIn[XUD_EP_COUNT_IN] =   {XUD_EPTYPE_CTL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL, XUD_EPTYPE_BUL};
+
```

Comment:
Quite a few main.xc files `#include "test_main.xc"` and use its common functions.  This file does not.  Is that intentional or an oversight?